tests/validationapp.py

```diff
@@ -1,12 +1,14 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+import json
+
 from pyramid.config import Configurator
 from pyramid.httpexceptions import HTTPBadRequest
 
 from cornice import Service
-from cornice.tests.support import CatchErrors
-import json
+
+from .support import CatchErrors
```

Comment:
Yes, and if it would be the case, it should live in some `cornice.testing` module